frontend/src/lib/components/apps/editor/settingsPanel/inputEditor/StaticInputEditor.svelte

```diff
@@ -24,17 +24,49 @@
 	import EditableSchemaDrawer from '$lib/components/schema/EditableSchemaDrawer.svelte'
 	import AppPicker from '$lib/components/wizards/AppPicker.svelte'
 
-	export let componentInput: StaticInput<any> | undefined
+	export let componentInput: (StaticInput<any> & { isLabeled?: boolean }) | undefined
 	export let fieldType: InputType | undefined = undefined
 	export let subFieldType: InputType | undefined = undefined
 	export let selectOptions: StaticOptions['selectOptions'] | undefined = undefined
 	export let placeholder: string | undefined = undefined
 	export let format: string | undefined = undefined
 	export let id: string | undefined
+	export let hasLabeledMode: boolean = false
 
 	const { onchange } = getContext<AppViewerContext>('AppViewerContext')
 
 	$: componentInput && onchange?.()
+
+	onMount(() => {
```

Comment:
is the onMount necessary? Do you need to wait for the dom to be mounted?

Reply:
It is a way to only call handleComponentInputChange once (on <StaticInputEdtitor /> mount)